cmd/lncli/cmd_open_channel.go

```diff
@@ -406,7 +406,7 @@ func openChannel(ctx *cli.Context) error {
 	if ctx.IsSet("utxo") {
 		utxos := ctx.StringSlice("utxo")
 
-		outpoints, err := utxosToOutpoints(utxos)
```

Comment:
I think the order of this commit and the next one can be swapped as this commit here prepares the `lncli` commit.

Reply:
done